src/Http/src/Config/HttpConfig.php

```diff
@@ -51,4 +52,16 @@ public function getMiddleware(): array
     {
         return $this->config['middleware'] ?? $this->config['middlewares'];
     }
+
+    /**
+     * If chunkSize isn't provided - using default values
+     */
+    public function getChunkSize(): ?int
+    {
+        if (\is_null($this->config['chunkSize']) || (int) $this->config['chunkSize'] < 0) {
+            return null;
+        }
+
```

Comment:
Maybe do you replace `(int) $this->config['chunkSize'] < 0` with `assert((int) $this->config['chunkSize'] > 0, 'Chunk size should be greater than 0.')` ?

Reply:
Chunk size can be equal 0